### PR TITLE
More small improvements

### PR DESCRIPTION
**Pull request by crotger**
Added browsing breadcrumbs to the top bar, added another field view to annotations, preparing for layout changes to the list pages.

### CODE REVIEW

File: views.py [hunk 1]
```diff
@@ -176,6 +206,8 @@ def page_detail(request, page_id, book_id=None):
     context['prev_pid'] = prev_pid
     context['next_pid'] = next_pid
 
+    context['breadcrumbs'][len(context['breadcrumbs']) - 1]['name'] = "Image " + page_json['rel_has_pagination_ssim'][0]
```
Comment (bcail) [inline views.py]: try context['breadcrumbs'][-1]['name']

File: views.py [hunk 2]
```diff
@@ -394,6 +442,9 @@ def print_detail(request, print_id):
         curr_annot = get_annotation_detail(annotation)
         context['annotations'].append(curr_annot)
 
+
+    context['breadcrumbs'][len(context['breadcrumbs']) - 1]['name'] = breadcrumb_detail(context, view="print")
```
Comment (bcail) [inline views.py]: context['breadcrumbs'][-1]

File: views.py [hunk 3]
```diff
@@ -421,6 +472,8 @@ def biography_detail(request, trp_id):
     prints_merged[len(prints_merged):] = prints_search
 
     context['prints'] = prints_merged
+
+    context['breadcrumbs'][len(context['breadcrumbs']) - 1]['name'] = breadcrumb_detail(context, view="bio")
```
Comment (bcail) [inline views.py]: context['breadcrumbs'][-1]

File: views.py [hunk 4]
```diff
@@ -128,6 +156,8 @@ def page_detail(request, page_id, book_id=None):
     context['lowres_url']="https://%s/fedora/objects/%s/datastreams/lowres/content" % (BDR_SERVER, page_pid)
     context['det_img_view_src']="https://%s/viewers/image/zoom/%s" % (BDR_SERVER, page_pid)
 
+    context['breadcrumbs'][len(context['breadcrumbs']) - 2]['name'] = breadcrumb_detail(context, view="print")
```
Comment (bcail) [inline views.py]: context['breadcrumbs'][-2]

File: views.py [hunk 5]
```diff
@@ -52,25 +66,38 @@ def book_list(request):
...
     return render(request, 'rome_templates/book_list.html', context)
 
 
 def book_detail(request, book_id):
     book_list_page = request.GET.get('book_list_page', 1)
-    context = std_context()
+    context = std_context(request.path_info)
     context['back_to_book_href'] = u'%s?page=%s' % (reverse('books'), book_list_page)
     context['book'] = Book.get_or_404(pid="%s:%s" % (PID_PREFIX, book_id))
+    context['breadcrumbs'][len(context['breadcrumbs']) - 1]['name'] = breadcrumb_detail(context)
```
Comment (bcail) [inline views.py]: context['breadcrumbs'][-1]

File: views.py [hunk 6]
```diff
@@ -20,7 +20,20 @@
...
 
 
-def std_context(style="rome/css/content.css",title="The Theater that was Rome"):
+def std_context(path, style="rome/css/content.css",title="The Theater that was Rome"):
+    pathparts = path.split(u'/')
+    breadcrumbs = []
+    url = "/"
+
+    for node in pathparts:
+        if(len(node) > 0):
```
Comment (bcail) [inline views.py]: if node:

**Comment by crotger** [comment]
Fixed those issues and recommitted

**Pull request merged**